tests/chainer_tests/functions_tests/math_tests/test_clip.py

```diff
@@ -62,6 +67,24 @@ def test_backward_cpu(self):
     def test_backward_gpu(self):
         self.check_backward(cuda.to_gpu(self.x), cuda.to_gpu(self.gy))
 
+    def check_double_backward(self, x_data, y_grad, gx_grad):
+        def f(x):
+            y = functions.clip(x, self.x_min, self.x_max)
+            return y * y
+
+        gradient_check.check_double_backward(
+            f, x_data, y_grad, gx_grad, dtype=numpy.float64)
+
+    @condition.retry(5)
```

Comment:
Please do not use `retry`. It can hide serious errors.

Reply:
Okay, I'll change the tolerances instead of using retry. More specifically, what types of errors do you mean and does this only apply to double backward tests?

Reply:
No, it doesn't only apply to double backward.
Various kind of errors are possible. One example is that the backward computation could be implemented incorrectly only in limited range of input domain.